index.html

```diff
@@ -98,6 +101,6 @@ <h2>Connect</h2>
   </section>
 
   <img src ="DGM.jpg" alt="A photo of the website creator.">
-
+  <script src="index.js"></script>
```

Comment:
nit: this works. the assignment seems to ask to keep your index.js file in a folder called js. This is fine now since it works, but later on, please 
- create the folder
- move the index.js file there
- change the src reference to `<script src="js/index.js"></script>`